tests/reportArticle.test.js

```diff
@@ -0,0 +1,78 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../src/index';
+import status from '../src/helpers/constants/status.codes';
+
+const { expect } = chai;
+chai.use(chaiHttp);
+let UserToken = '';
+
+describe('testing the report article endpoint', () => {
+  it('should login a user and get a token to use against', (done) => {
+    chai
+      .request(app)
+      .post('/api/users/login')
+      .send({
+        email: 'alain1@gmail.com',
+        password: 'password23423'
+      })
+      .end((err, res) => {
+        UserToken = res.body.user.token;
+        done();
+      });
+  });
+
+  it('should report an article', (done) => {
+    const slug = 'What-is-a-Version-1-UUID';
+    chai
+      .request(app)
+      .post(`/api/articles/${slug}/report`)
+      .set('authorization', UserToken)
+      .send({
+        reason: 1
+      })
+      .end((err, res) => {
+        expect(res.status).eql(status.CREATED);
```

Comment:
Why use ``status`` while in the ``articleController`` you used ``statusCodes``let us discuss this in our meetups

Reply:
didn't wanna repeat the same mistake, my brother!!?

Reply:
stay consistent change both of them